gulp/tasks/scripts.js

```diff
@@ -189,8 +188,6 @@ function compileScriptsBabel() {
     return gulp.src(`${config.src}/**/*.js`, {
         // since: gulp.lastRun('compileScriptsBabel')
     })
-    // .pipe(pi.sourcemaps.init({loadMaps: true}))
-    .pipe(pi.sourcemaps.init())
```

Comment:
Are JS & CSS sourcemaps being removed entirely or just to another place? At a glance I don't see them being re-added anywhere below. (maybe they're in `gulp/tasks/webpack.js`?)

I think sourcemaps should continue to be included because:

1. our code is opensource
2. it makes debugging production problems easier
3. it allows for code coverage if you run tests in a browser (like a11y tests)

Reply:
Sourcemaps were not working after adding webpack as far as I was able to tell. Maybe I don't know what I'm looking for. Do you see evidence of sourcemaps on qa?